src/main/java/com/jaoafa/mymaid4/lib/EBan.java

```diff
@@ -150,7 +151,7 @@ public Result addBan(String banned_by, String reason) {
                         player.getPlayer().setGameMode(GameMode.CREATIVE);
                     }
 
-                    player.getPlayer().teleport(MyMaidData.paradiseLocation);
+                    MyMaidLibrary.teleportToParadise((Player) player, null);
```

Comment:
TeleportCauseをJail/EBan適用時だけ付与するのも不自然なので、一律で `TeleportCause.PLUGIN` を渡してしまってもいいかもしれません。

```suggestion
                    MyMaidLibrary.teleportToParadise((Player) player, TeleportCause.PLUGIN);
```